lib/rubocop/cop/rails/create_table_with_timestamps.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks the migration for which timestamps are not included
+      # when creating a new table.
+      # In many cases, timestamps are useful information and should be added.
+      #
+      # @example
+      #   # bad
+      #   create_table :users
+      #
+      #   # bad
+      #   create_table :users do |t|
+      #     t.string :name
+      #     t.string :email
+      #   end
+      #
+      #   # good
+      #   create_table :users do |t|
+      #     t.string :name
+      #     t.string :email
+      #
+      #     t.timestamps
+      #   end
+      class CreateTableWithTimestamps < Cop
+        MSG = 'Add timestamps when creating a new table.'.freeze
+
+        def_node_matcher :create_table_with_block?, <<-PATTERN
+          (block
+            (send nil? :create_table ...)
+            (args (arg _var))
+            begin)
```

Comment:
The last `begin` matcher is not correct. This pattern does not match empty blocks or blocks that have one statement only.
For example, this cop should add offenses for the following code, but it does not add:

```ruby
create_table :users do |t|
end


create_table :users do |t|
  t.string :name
end
```


Because the ASTs for this cases don't have `begin` node.


```console
$ ruby-parse test.rb
(begin
  (block
    (send nil :create_table
      (sym :users))
    (args
      (procarg0 :t)) nil)
  (block
    (send nil :create_table
      (sym :users))
    (args
      (procarg0 :t))
    (send
      (lvar :t) :string
      (sym :name))))
```